model/src/main/java/org/mskcc/cmo/metadb/model/MetaDbRequest.java

```diff
@@ -131,6 +138,25 @@ public MetaDbProject getMetaDbProject() {
     public void setMetaDbProject(MetaDbProject metaDbProject) {
         this.metaDbProject = metaDbProject;
     }
+    
+    public List<RequestMetadata> getRequestMetadataList() {
+        return requestMetadataList;
+    }
+
+    public void setRequestMetadataList(List<RequestMetadata> requestMetadataList) {
+        this.requestMetadataList = requestMetadataList;
+    }
+    
+    /**
+     * 
```

Comment:
if adding javadoc comments then a brief description of what the method does should be added as well